app/views/home/index.html.erb

```diff
@@ -2,28 +2,19 @@
 
 
 
-<% if @post %>
-<%= link_to 'My Posts', @post,:type => "button", :class => "btn btn-primary btn-sm", :style => ''%>
+<% if @posts %>
+<%= link_to 'My Posts', posts_path,:type => "button", :class => "btn btn-primary btn-sm", :style => ''%>
```

Comment:
remove the style => "" 
And you case the latest syntax
e.g above can be converted to 
```
<%= link_to 'My Posts', posts_path,type: "button", class: "btn btn-primary btn-sm", style: ''%>
```
And you can see latest one is easier to write

Reply:
OK, I will follow this convention in my next tasks.